### PR TITLE
scylla-python3: move scylla-python3 to separated repository

### LINKED ISSUE #6751: dist: scylla-python3 should be separated repository
Except scylla-python3, each scylla package has its own git repository, same package script filename, same build directory structure.
To put python3 thing on scylla repo, we created 'python3' directory on multiple locations, made '-python3' suffixed files, dig deeper build directory not to conflict scylla-server package build.
We should move all scylla-python3 related files to new repository, scylla-python3.
Since we recently added scylla-jmx and scylla-tools as git submodule, we can move scylla-python3 to another submodule, too.

Here's RFC version of scylla-python3 repo, with latest next code:
https://github.com/syuu1228/scylla-python3

After moved it to new repo, we can drop these scripts:
```
[syuu@monster scylla.3]$ git status
On branch next
Your branch is up to date with 'origin/next'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	deleted:    dist/debian/python3/build_deb.sh
	deleted:    dist/debian/python3/changelog.template
	deleted:    dist/debian/python3/control.template
	deleted:    dist/debian/python3/debian/compat
	deleted:    dist/debian/python3/debian/copyright
	deleted:    dist/debian/python3/debian/rules
	deleted:    dist/debian/python3/debian_files_gen.py
	deleted:    dist/redhat/python3/build_rpm.sh
	deleted:    dist/redhat/python3/python.spec
	deleted:    python3/install.sh
	deleted:    reloc/python3/build_deb.sh
	deleted:    reloc/python3/build_reloc.sh
	deleted:    reloc/python3/build_rpm.sh
	deleted:    scripts/create-relocatable-package-python3.py
```

### PR DESCRIPTION
**Pull request by syuu1228**
Except scylla-python3, each scylla package has its own git repository, same package script filename, same build directory structure.
To put python3 thing on scylla repo, we created 'python3' directory on multiple locations, made '-python3' suffixed files, dig deeper build directory not to conflict scylla-server package build.
We should move all scylla-python3 related files to new repository, scylla-python3.

Fixes #6751

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -18,3 +18,6 @@
 [submodule "scylla-tools"]
 	path = scylla-tools
 	url = ../scylla-tools-java
+[submodule "scylla-python3"]
+	path = scylla-python3
```
Comment (avikivity) [inline .gitmodules:22]: Let's move it to tools/python3 instead (no need to repeat scylla). 

I also want to move scylla-jmx to tools/jmx and scylla-tools to tools/java.

Comment (avikivity) [reply .gitmodules:22]: What about the build system changes?

```
$ git grep python3 configure.py
configure.py:#!/usr/bin/env python3
configure.py:arg_parser.add_argument('--python', action='store', dest='python', default='python3',
configure.py:          command = ./reloc/python3/build_reloc.sh
configure.py:          command = ./reloc/python3/build_rpm.sh
configure.py:          command = ./reloc/python3/build_deb.sh
configure.py:        build build/release/scylla-python3-package.tar.gz: build-python-reloc
configure.py:        build dist-python-rpm: build-python-rpm build/release/scylla-python3-package.tar.gz
configure.py:        build dist-python-deb: build-python-deb build/release/scylla-python3-package.tar.gz
```

Comment (syuu1228) [reply .gitmodules:22]: > Let's move it to tools/python3 instead (no need to repeat scylla).
> 
> I also want to move scylla-jmx to tools/jmx and scylla-tools to tools/java.
I think we can create another directory for these submodules, something like "packages".
ex) packages/jmx, packages/tools, packages/python3...
I suppose it's hard to understand "tools/java" is scylla-tools submodule , and also we use tools directory for other purpose, better to use other place.

Comment (avikivity) [reply .gitmodules:22]: I think there won't be a problem. tools/ is a place for tools, like tools/toolchain, and we can add submodules and non-submodules there.

**Comment by syuu1228** [comment]
@avikivity moved submodules to tools/, apply build system changes on configure.py.
Please review again.

**Comment by avikivity** [comment]
Unmerged, next promotion fails since it tries to execute files in reloc/python3/. Please add scripts there to call the new locations.

**Pull request merged**